src/main/java/BadPracticesModel.java

```diff
@@ -0,0 +1,68 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class BadPracticesModel {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
+    // Unused variable
+    private String unusedVariable = "I am unused";
+
+    public boolean newMethod(String st1, String st2) {
+        // Using '==' for string comparison instead of 'equals'
+        if (st1 == st2) {
+            return true;
+        }
+        return false;
+    }
+
+    public void methodWithRedundantCode() {
+        // Redundant code and unused variable
+        String redundantString = "This is redundant";
```

Comment:
The variable 'redundantString' is declared but never used meaningfully. (Severity: low)